lib/awful/util.lua

```diff
@@ -512,7 +512,7 @@ function util.table.iterate(t, filter, start)
     return function ()
         while count < length do
             local item = t[index]
-            index = util.cycle(#t, index + 1)
+            index = gmath.cycle(#t, index + 1)
```

Comment:
@Veratil 
What do you recommend to to?
Should we get your PRs in first, or do you not mind rebasing them?
Some of them are not green?!